### PR TITLE
General Settings ACL Control

### PR DESCRIPTION
**Pull request by NCommander**
Resolves: #2273

Allows limited access to the admin panel for registered users on the platform on ACL controls.

### CODE REVIEW

File: backend/globaleaks/utils/utility.py [hunk 1]
```diff
@@ -82,7 +82,7 @@ def get_disk_space(path):
 
 
 def read_file(p):
-    with open(p, 'r') as f:
```
Comment (evilaliv3) [inline backend/globaleaks/utils/utility.py]: @NCommander: please keep commits separated where possible and anticipate patches to devel when they are relevant to stabilization. :)

Comment (evilaliv3) [reply backend/globaleaks/utils/utility.py]: The fix was not compatible with python 2.7

Proper solution compatibile with both 2.7 and 3 is to use io.open that is backported to python2.7

https://github.com/globaleaks/GlobaLeaks/commit/4e2ddee31a57ce2229c2615191a4148c84a05c2d

File: backend/requirements/requirements-win32.txt [hunk 2]
```diff
@@ -1,23 +1,22 @@
...
 pycparser==2.14
 python_debian==0.1.28
-python_gnupg==0.3.8
+python_gnupg==0.4.2
 requests==2.9.1
 scrypt==0.8.6
 service_identity==16.0.0
 six==1.10.0
-sqlalchemy==1.0.11
-txtorcon==0.15.1
```
[the remark this answers is not in the document]
Comment (NCommander) [reply backend/requirements/requirements-win32.txt]: GOt left in by accident, shouldn't have been in the PR. Oops

**Comment by evilaliv3** [comment]
Review applied and pull request ready for merge!

Thanks @NCommander !!

**Pull request merged**